ros2/cc_defs.bzl

```diff
@@ -123,7 +124,8 @@ def ros2_cpp_test(name, ros2_package_name = None, set_up_ament = False, **kwargs
         name: A unique target name.
         ros2_package_name: If given, defines a ROS package name for the target.
             Otherwise, the `name` is used as the package name.
-        set_up_ament: If true, sets up ament file tree for the test target.
+        set_up_ament: If true, wrap the test target to set up ament file tree and
```

Comment:
Please rephrase here and for the corr. Python macro as e.g.

```
If true, a custom launcher is generated that sets up the following for the test target:
* the ament file tree
* ROS_HOME and ROS_LOG_DIR are set to TEST_UNDECLARED_OUTPUTS_DIR, if available, otherwise to TEST_TMPDIR. See [here](https://bazel.build/reference/test-encyclopedia#initial-conditions) for more info.
```